src/test/java/com/swift/RegistryTest.java

```diff
@@ -21,6 +22,9 @@
 public class RegistryTest {
 
     private final static Registry registry;
+    private static String ignoredListRegistryFormatExampleMethod [] = {"IQ", "SC", "CR", "LC", "ST", "BY"};
```

Comment:
We want to use a List<String> over a String[] so that we can use the `contains()` method 

We can define a List statically so that we don't have to build it at runtime 

`private static List<String> ignoredCountryCodesForRegistryFormatExample = Arrays.asList("IQ", "SC", ...);`

However, we don't care about the order of the country codes and they should be unique so we should use a Set instead of a list

`private static Set<String> ignoredCountryCodesForRegistryFormatExample = new HashSet<>(Arrays.asList("IQ", "SC", ...));`

Reply:
Thank you. Learnt about Array, Set and Lists. Tests modified in the next commit.